src/amo/reducers/collections.js

```diff
@@ -0,0 +1,160 @@
+/* @flow */
+import { createInternalAddon } from 'core/reducers/addons';
+import type { AddonType, ExternalAddonType } from 'core/types/addons';
+
+export const FETCH_COLLECTION = 'FETCH_COLLECTION';
+export const LOAD_COLLECTION = 'LOAD_COLLECTION';
+
+export type CollectionType = {
```

Comment:
I did not put strict here because Flow complains in the reducer (when I spread the state..)

Reply:
> I did not put strict here because Flow complains in the reducer (when I spread the state..)

Ugh, yeah, I still have that issue in my awesomebar history https://github.com/facebook/flow/issues/2405

Reply:
You could possibly keep this exact and then import `$Shape` from `src/core/types/util.js` and use `$Shape<CollectionType>` only where you need it for spreads. If that's a hassle then it's no biggie.